client_it_test.go

```diff
@@ -994,7 +994,7 @@ func TestRowRangeQuery(t *testing.T) {
 
 func TestRangeField(t *testing.T) {
 	client := getClient()
-	field := index.Field("rangefield", OptFieldTypeInt(10, 20))
+	field := index.Field("rangefield", OptFieldTypeInt())
```

Comment:
We don't need it yet, but we should possibly at some point test things like the enforcement of min/max (like what happens if you submit values outside the min/max range). Only it occurs to me I don't know what results we'd expect so I think testing it might be premature.

Reply:
I wasn't able to return an error since the `FieldOption` interface is public, but updated `OptFieldTypeInt` to panic if `min > max`.